### PR TITLE
Fix typos and wording

### LINKED ISSUE #13: Typos in the source code
I'd like to contribute some more and saw some typo's in the sourcecode.

Would it be okay if I fixed those and send you a pull request?

All in one PR or should I create a separate one per file? Or any other suggestion you might have.

### PR DESCRIPTION
**Pull request by starquake**
Fixes #13

### CODE REVIEW

File: src/command_line.rs [hunk 1]
```diff
@@ -273,8 +273,8 @@ impl Config {
             Arg::Long("ignoregroup") => self.ignore_group.push(value?.to_string()),
             Arg::Long("arch") => self.arch = Some(value?.to_string()),
             Arg::Long("color") => self.color = Colors::from(value.unwrap_or("always")),
-            Arg::Long(a) if !arg.is_pacman_arg() => bail!("unkown option --{}", a),
-            Arg::Short(a) if !arg.is_pacman_arg() => bail!("unkown option -{}", a),
+            Arg::Long(a) if !arg.is_pacman_arg() => bail!("unknwn option --{}", a),
```
Comment (Morganamilo) [inline src/command_line.rs:276]: Not only I make typos ;)

**Comment by Morganamilo** [comment]
That's a lot of typos. One issue with auto complete is you do it once and all of a sudden it's everywhere. Anyway thanks for fixing all these. Looks good apart from the one typo.

File: src/command_line.rs (hunk 1 above)
Comment (starquake) [reply src/command_line.rs:276]: Haha oops.

**Comment by starquake** [comment]
Well that was silly. Fixed the typo.

**Pull request merged**